src/main/target/OMNIBUSF4/target.c

```diff
@@ -40,19 +40,19 @@ const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
     DEF_TIM(TIM8,  CH3, PC8,  TIM_USE_PWM,                 TIMER_OUTPUT_NONE,     0), // S5_IN
     DEF_TIM(TIM8,  CH4, PC9,  TIM_USE_PWM,                 TIMER_OUTPUT_NONE,     0), // S6_IN
 #endif // CL_RACINGF4
-    DEF_TIM(TIM3,  CH3, PB0,  TIM_USE_MOTOR,               TIMER_OUTPUT_STANDARD, 0), // S1_OUT D1_ST7
-    DEF_TIM(TIM3,  CH4, PB1,  TIM_USE_MOTOR,               TIMER_OUTPUT_STANDARD, 0), // S2_OUT D1_ST2
-    DEF_TIM(TIM2,  CH4, PA3,  TIM_USE_MOTOR,               TIMER_OUTPUT_STANDARD, 1), // S3_OUT D1_ST6
-    DEF_TIM(TIM2,  CH3, PA2,  TIM_USE_MOTOR,               TIMER_OUTPUT_STANDARD, 0), // S4_OUT D1_ST1
+    DEF_TIM(TIM1,  CH2N, PB0,  TIM_USE_MOTOR,               TIMER_OUTPUT_ENABLED|TIMER_OUTPUT_INVERTED, 0), // M1 - D2_ST6
```

Comment:
This changes timers for the other targets that are defined in this file. This should not be done if it is only a fix for CL_RACINGF4.

If the `target.c` file gets too complicated after this change, we can also separate it out into a target directory if its own.